modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionDeployWithVirtualInstanceId/build.gradle

```diff
@@ -0,0 +1,11 @@
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":client-extension:deploy"
+
+	doLast {
+		FileTree fileTree = project.fileTree("bundles/osgi/client-extensions/vi.localtest.me")
+
+		assert fileTree.contains(project.file("bundles/osgi/client-extensions/vi.localtest.me/client-extension.zip"))
```

Comment:
You could probably just reference the file directly here and test `exists()`. That would  save you having to reference the folder path twice.

Reply:
Something like this for example? 
```
String virtualInstanceId = "vi.localtest.me"
File clientExtensionZip = new File("${gradle.liferayWorkspace.homeDir}/osgi/client-extensions/${virtualInstanceId}/client-extension.zip")

assert clientExtensionZip.exists()
```

Reply:
Yup, exactly!

Side note: be sure to include a line break between statements where order matters. So you'd need a line break after the declaration of `String virtualInstanceId` since it is then used on the next line.